README.md

```diff
@@ -1,6 +1,21 @@
 GXUI - A Go cross platform UI library.
 =======
 
+Usage
+---
+After setting up ```GOPATH``` (see [Go documentation](https://golang.org/doc/code.html)), get the required Go libraries and this repository:
+
+    go get code.google.com/p/freetype-go/freetype
+    go get github.com/go-gl/gl/v3.2-core/gl
+    go get github.com/go-gl/glfw/v3.1/glfw
+    go get github.com/google/gxui
```

Comment:
Doing `go get github.com/google/gxui` would get all dependencies automatically, so it is a good idea to suggest fetching them manually?

Also, wouldn't it be a better default recommendation to use `-u` flag, as in `go get -u github.com/google/gxui`? That should reduce the chance of people having problems due to having older version of `gxui` or one of its dependencies that would be fixed if they used `-u` flag. Being only a suggestion in README, people can always avoid using `-u` flag if they desire.